frontend/rust-lib/flowy-database/Cargo.toml

```diff
@@ -16,8 +16,8 @@ r2d2 = "0.8.9"
 libsqlite3-sys = { version = ">=0.8.0, <0.24.0", features = ["bundled"] }
 scheduled-thread-pool = "0.2.5"
 error-chain = "=0.12.0"
-openssl = { version = "0.10.38", optional = true, features = ["vendored"] }
-openssl-sys = { version = "0.9.69", optional = true, features = ["vendored"] }
+openssl = { version = "0.10.38",  features = ["vendored"] }
+openssl-sys = { version = "0.9.69",  features = ["vendored"] }

```

Comment:
Using the optional flag causes the android build to fail completely. It throws an error saying can't find openssl.